docs/source/usingmame/defaultkeys.rst

```diff
@@ -1,106 +1,558 @@
+**4**
+    Player 4 start or 4 players mode
+**Backspace**
+    Bill 1 (For machines with a bill receptor, typically gambling)
+**9**
+    Service 1 (Service buttons are typically used to give free credits or to
+    navigate the internal operator service menus)
+**0**
+    Service 2
+**-** (not numeric keypad)
+    Service 3
+**=** (not numeric keypad)
+    Service 4
+**Ctrl**/**Mouse B0**/**Gun 1 Button 0**
+    Mahjong Kan
+**Alt**/**Mouse B2**/**Gun 1 Button 1**
+    Mahjong Pon
+**Spacebar**/**Mouse B1**/**Joystick 1 Button 1**
+    Mahjong Chi
+**Shift**
+    Mahjong Reach
+**Z**
+    Mahjong Ron
+**A**
+    Player 1 Mahjong/Hanafuda A
+**B**
+    Player 1 Mahjong/Hanafuda B
+**C**
+    Player 1 Mahjong/Hanafuda C
+**D**
+    Player 1 Mahjong/Hanafuda D
+**E**
+    Player 1 Mahjong/Hanafuda E
+**F**
+    Player 1 Mahjong/Hanafuda F
+**G**
+    Player 1 Mahjong/Hanafuda G
+**H**
+    Player 1 Mahjong/Hanafuda H
+**I**
+    Player 1 Mahjong I
+**J**
+    Player 1 Mahjong J
+**K**
+    Player 1 Mahjong K
+**L**
+    Player 1 Mahjong L
+**M**
+    Player 1 Mahjong M
+**N**
+    Player 1 Mahjong N
+**O**
+    Player 1 Taiwanese Mahjong O
+**P**
+    Player 1 Taiwanese Mahjong P
+**Q**
+    Player 1 Taiwanese Mahjong Q
+**Right Alt**
+    Player 1 Mahjong/Hanafuda Last Chance
+**Right Ctrl**
+    Player 1 Mahjong/Hanafuda Take Score
+**Right Shift**
+    Player 1 Mahjong/Hanafuda Double Up
+**Y**
+    Player 1 Mahjong/Hanafuda Flip/Flop
+**Enter**
+    Player 1 Mahjong/Hanafuda Big
+**Backspace**
+    Player 1 Mahjong/Hanafuda Small
+**9**
+    Service 1 (Service buttons are typically used to give free credits or to
+    navigate the internal operator service menus)
+**0**
+    Book-Keeping
+
+
+.. _default-gambling-keys:
+
+Default Gambling Keys
+=====================
+
+All the keys below are fully configurable in the user interface. This list shows
+the standard keyboard configuration.
+
+
+**5**
+    Coin slot 1
+**6**
+    Coin slot 2
+**7**
+    Coin slot 3
+**8**
+    Coin slot 4
+**1**
+    Player 1 start or 1 player mode
+**2**
+    Deal
+**3**
+    Double Up
+**4**
+    Take Score
+**Backspace**
+    Bill 1 (For machines with a bill receptor, typically gambling)
+**9**
+    Service 1 (Service buttons are typically used to give free credits or to
+    navigate the internal operator service menus)
+**0**
+    Service 2
+**-** (not numeric keypad)
+    Service 3
+**=** (not numeric keypad)
+    Service 4
+**L**
+    Stand
+**M**
+    Bet
+**A**
+    High
+**D**
+    Half Gamble
+**I**
+    Payout
+**O**
+    Door
+**Q**
+    Key In
+**W**
+    Key Out
+**S**
+    Low
+**9**
+    Service
+**0**
+    Book-Keeping
+
+.. _default-poker-keys:
+
+Default Poker Keys
+==================
+
+All the keys below are fully configurable in the user interface. This list shows
+the standard keyboard configuration.
+
+
+**Z**
+    Hold 1
+**X**
+    Hold 2
+**C**
+    Hold 3
+**V**
+    Hold 4
+**B**
+    Hold 5
+**N**
+    Cancel
+**1**
+    Bet
```

Comment:
Not related to this directly, but why do we even have this?  It conflicts with the Start key.  We should probably just get rid of it and make poker games use the regular Bet input.

Reply:
Great question that I don't have a great answer for. If this is going to happen, let me know so I can do the adjustments needed for the docs.